src/misc/ErrorsPanel/ErrorsPanel.js

```diff
@@ -25,12 +25,12 @@ export const ErrorsPanel = (props) => {
   }
 
   return (
-    <Paper className={classes.errorsContainer}>
-      <Typography className={classes.errorsHeader}>
+    <Paper className={classes.errorsContainer} data-cy="errors-panel">
+      <Typography className={classes.errorsHeader} data-cy="errors-header">
         {labels.mainError} {errorsCountLabel}
       </Typography>
       {errors.map((error, index) => (
-        <Accordion key={'error' + index}>
+        <Accordion key={'error' + index} data-cy="error-accordion" data-cy-index={'error-accordion-' + index}>
```

Comment:
Do we really need both `data-cy` and `data-cy-index` properties ?
If we only set `data-cy={'error-accordion-' + index}`, it seems we can still select all elements whose data-cy starts with "error-accordion" with a selector like `cy.get('[data-cy^=error-accordion]')` (I have not tested it yet though)